lib/broadway_rabbitmq/amqp_client.ex

```diff
@@ -175,6 +175,22 @@ defmodule BroadwayRabbitMQ.AmqpClient do
       the `:declare` and `:bindings` options (described above).
       """
     ],
+    consume_options: [
+      type: :keyword_list,
+      default: [],
+      doc: """
+      Options passed down to `AMQP.Basic.consume/4`. Not all options supported by
+      `AMQP.Basic.consume/4` are available here as some options would conflict with
+      the internal implementation of this producer.
```

Comment:
We support all except the `no_wait` option as of now, confirm?

Reply:
Yes exactly!